spacksites/spack-env-templates/slurm/gpu_first.yaml

```diff
@@ -0,0 +1,20 @@
+spack:
+
+  specs:
+  # CUDA packages, compile them with gcc@12
+  - matrix:
+    - - 'cuda@12.2.2' # v12.2.2 matches the latest CUDA installed on Young
+      #- 'namd@2.14 +cuda cuda_arch=60,70,80' # requires cuda < 12
+      - 'py-alphafold@2.2.4 +cuda cuda_arch=60,70,80 ^py-scipy@1.11.4 ^py-pythran@0.12.2 ^openmm@7.5.1 +cuda cuda_arch=60,70,80'
+      # kokkos will require only one cuda_arch as different bit sizes for some types betw 60, 70 and 80 - can use the lowest
+      - gromacs@2024.3 +cuda cuda_arch=60,70,80 ^openmpi@4.1.6 +cuda
+      - gromacs@2023.5 +cuda cuda_arch=60,70,80 +plumed ^openmpi@4.1.6 +cuda
+      - openmpi@4.1.6 +cuda cuda_arch=60,70,80 fabrics=psm2,cma,ucx,ofi schedulers=sge
+    - - '%gcc@12.3.0'
+  # nvhpc packages
+  #- 'vasp@6.4.3 %nvhpc +cuda cuda_arch=60,70,80 +hdf5'
+  concretizer:
+    unify: when_possible
+  view: false
+  include_concrete:
+  - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/base
```

Comment:
This is the absolute path of the environment I'm currently using, but I don't know if you want to retain this name or rather using a more meaningful one (I copied `2025-02` from https://github.com/UCL-ARC/hpc-spack/issues/64#issuecomment-2711378522 before realising I could have picked a different month......